manticore/core/workspace.py

```diff
@@ -401,7 +410,8 @@ def __init__(self, desc=None):
         :param desc: A descriptor ('type:uri') of where to write output.
         """
         self._named_key_prefix = 'test'
-        self._store = _create_store(desc)
+        self._descriptor = desc
```

Comment:
Can we simplify this `self._descriptor` vs `self.descriptor` stuff? Can't we just do `self.descriptor = desc` and be done? I don't see a reason to have a property that computes that on every access, or even recompute it once from the `_store`, since the end result should be the same as `desc`?